lib/protobuf/protoc/generator/message.ex

```diff
@@ -76,15 +76,22 @@ defmodule Protobuf.Protoc.Generator.Message do
     inspect(exts)
   end
 
-  def msg_opts_str(%{syntax: syntax}, opts) do
+  def msg_opts_str(%{syntax: syntax, custom_field_options?: custom_field_options}, opts) do
     msg_options = opts
 
     opts = %{
       syntax: syntax,
       map: msg_options && msg_options.map_entry,
-      deprecated: msg_options && msg_options.deprecated
+      deprecated: msg_options && msg_options.deprecated,
     }
 
+    opts =
+      if custom_field_options do
+        Map.put(opts, :custom_field_options?, true)
```

Comment:
Is it bad to also put the field when `false`?

Reply:
added